code/modules/projectiles/guns/rifles.dm

```diff
@@ -714,10 +714,11 @@
 		/obj/item/attachable/angledgrip,
 		/obj/item/attachable/flashlight,
 		/obj/item/attachable/bipod,
-		/obj/item/attachable/stock/rifle,
 		/obj/item/attachable/compensator,
 		/obj/item/attachable/magnetic_harness,
 		/obj/item/attachable/scope,
+		/obj/item/attachable/scope/marine,
+		/obj/item/attachable/scope/mini,
```

Comment:
commenting for clarification due to balance freeze, those dont fall into balance wise because theyre meant to fit scopes fine, just whoever set those thought it would also allow in subtypes